org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/managers/GradleProjectImporter.java

```diff
@@ -186,14 +213,32 @@ public void importToWorkspace(IProgressMonitor monitor) throws CoreException {
 				JavaLanguageServerPlugin.logException("Failed to update digest for gradle build file", e);
 			}
 		});
+		for (IProject gradleProject : ProjectUtils.getGradleProjects()) {
+			gradleProject.deleteMarkers(COMPATIBILITY_MARKER_ID, true, IResource.DEPTH_INFINITE);
```

Comment:
gradleProject.deleteMarkers(COMPATIBILITY_MARKER_ID, true, ~~IResource.DEPTH_INFINITE~~ IResource.DEPTH_ZERO);

Reply:
fixed in https://github.com/eclipse/eclipse.jdt.ls/pull/1965/commits/87298b495b093411006826cc69f1775704fea69d